libs/core/src/lib/avatar/avatar.component.ts

```diff
@@ -70,14 +78,30 @@ export class AvatarComponent implements OnChanges, OnInit, CssClassBuilder {
     /** A number from 1 to 10 representing the background color of the Avatar. */
     @Input() colorAccent: ColorAccent = null;
 
+    /**
+    * @deprecated
+    * Deprecated according to Rename `backgroundImage` to `image`
+    * */
     /** Background image url. */
     @Input()
     backgroundImage: string = null;
 
+    /** Background image resource: url or base64. */
+    @Input()
+    set image(value: string) {
+        if (value) {
+            this._image = 'url(' + value + ')';
+        } else if (this.backgroundImage) {
```

Comment:
I don't understand why is this `else if` part needed.
If someone is setting the image using `[image]` but has previously used `[backgroundImage]`, then use previously passed `[backgroundImage]` instead?